RecoEgamma/EgammaTools/python/calibratedEgammas_cff.py

```diff
@@ -11,12 +12,15 @@
                                     recHitCollectionEE = cms.InputTag('reducedEcalRecHitsEE'),
                                     produceCalibratedObjs = cms.bool(True)
                                    )
+from Configuration.Eras.Modifier_run2_egamma_2017_cff import run2_egamma_2017
+run2_egamma_2017.toModify(calibratedEgammaSettings,correctionFile = _correctionFile2017UL)
+
+from Configuration.Eras.Modifier_run2_egamma_2018_cff import run2_egamma_2018
+run2_egamma_2018.toModify(calibratedEgammaSettings,correctionFile = _correctionFile2018UL)
```

Comment:
```suggestion
from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
(run2_miniAOD_UL & run2_egamma_2017).toModify(calibratedEgammaSettings,correctionFile = _correctionFile2017UL)

from Configuration.Eras.Modifier_run2_egamma_2018_cff import run2_egamma_2018
(run2_miniAOD_UL & run2_egamma_2018).toModify(calibratedEgammaSettings,correctionFile = _correctionFile2018UL)
```

the updates are expected to be only for UL setup in the backport.
IIUC, in this case, without my proposed change, this PR will override the behavior for `run2_miniAOD_94XFall17`